src/backend/distributed/connection/placement_connection.c

```diff
@@ -47,17 +49,14 @@ typedef struct ConnectionReference
 	bool hadDML;
 	bool hadDDL;
 
-	/*
-	 * Membership in ConnectionPlacementHashEntry->connectionReferences or
-	 * ColocatedPlacementsHashEntry->connectionReferences.
-	 */
-	dlist_node placementNode;
```

Comment:
To simplify the implementation of `GetPlacementListConnection`. There is no reason to keep track of all connections that accessed a placement right now.

I guess we could still shove it in a list and not use it...